tests/topotests/lib/topogen.py

```diff
@@ -492,7 +492,15 @@ def stop_topology(self):
                 "Errors found post shutdown - details follow: {}".format(errors)
             )
 
-        self.net.stop()
+        try:
+            # Stop toplogy and remove tmp files
+            self.net.stop()
+
+        except OSError:
+            # OSError exception is raised when mininet tries to stop switch
+            # though switch is stopped once but mininet tries to stop same
+            # switch again, where it ended up with exception
```

Comment:
Can we log something also in such a case?

Reply:
> Can we log something also in such a case?

Hi Donatas Abraitis,

Addressed review comment.
Thank you.

Regards,
Bharath